src/track/cue.h

```diff
@@ -39,6 +39,7 @@ class Cue : public QObject {
             double samplePosition = kNoPosition);
     void setEndPosition(
             double samplePosition = kNoPosition);
+    void shiftPosition(double frameOffset);
```

Comment:
Renaming suggested: shiftPositionFrames (because *cue positions* are measured in *stereo samples*).